src/main/java/com/amazon/checkerframework/cryptopolicy/CryptoPolicyComplianceVisitor.java

```diff
@@ -201,13 +197,16 @@ private boolean hasSuppresssCryptoAnnotation(final Element elt, final String sup
             // of validation. Instead, we just check if the string is a valid URL to deter users
             // from cheating by putting in empty string, etc.
             try {
-                final URL issueURL = new URL(anno.issue());
+                final URL issueUrl = new URL(anno.issue());
+                System.out.println("Suppressing warning for "
```

Comment:
Is this `println()` supposed to be here? It seems at best unrelated to this change.

Reply:
That's unrelated but useful. The idea is that you can grep the build log to see if warnings were suppressed.

Reply:
Probably should be a separate CR but it's fine.